PerkTutorCouchDB/PerkTutorCouchDB.py

```diff
@@ -530,7 +530,7 @@ def getNodeDefaultWriteTypeExtension( node ):
     fileWriterFileType = ioManager.fileWriterFileType( node )
     fileWriterDefaultExtension = ioManager.fileWriterExtensions( node )[ 0 ]
 
-    extensionRegex = "\.\\w+\\b" # Search for everything after the first . until the next non-alphanumeric character
+    extensionRegex = "\\.\\w+\\b" # Search for everything after the first . until the next non-alphanumeric character

```

Comment:
`pyupgrade` automatically made this change. Please make sure this is acceptable for your intended regex.

Reply:
Thanks, confirmed that it works.